msteams-platform/sbs-botbuilder-taskmodule.yml

```diff
@@ -85,6 +85,7 @@ items:
        You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
     
     1. If you've created a new resource group, select the required location from **New resource group location** dropdown list.
+       For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
remove the text from this line and add it as part of line 85. Follow the suggestions mentioned in other files and implement the same here.

Reply:
The link is removed as suggested.